api/lambdas/cda-to-visualization/index.js

```diff
@@ -0,0 +1,224 @@
+const puppeteer = require("puppeteer-core");
+const chromium = require("@sparticuz/chromium");
+const AWS = require("aws-sdk");
+const SaxonJS = require("saxon-js");
+const fs = require("fs");
+const styleSheetText = require("./stylesheet.js");
+
+const getEnvOrFail = name => {
+  const value = process.env[name];
+  if (!value || value.trim().length < 1) throw new Error(`Missing env var ${name}`);
+  return value;
+};
+
+const bucketName = getEnvOrFail("MEDICAL_DOCUMENTS_BUCKET_NAME");
+
+const s3client = new AWS.S3({
+  signatureVersion: "v4",
+});
+
+module.exports = async req => {
+  const { fileName, conversionType } = req;
+
+  const { document, contentType } = await downloadDocumentFromS3({ fileName });
+
+  if (contentType === "application/xml" || contentType === "text/xml") {
```

Comment:
nbd for now - but really we should be checking this in the API before we even hit the lambda, and erroring if the user tries to convert a doc that's not XML

if we're agreed - can we raise an issue for this so we don't forget?

Reply:
Ill update this, shouldnt be too long